docs/vgclog.md

```diff
@@ -0,0 +1,269 @@
+
+For example, consider the garbage collector for the `gencon` policy, which uses partial cycles and global cycles. The partial cycle consists of just 1 GC operation, scavenge, that runs on the *nursery* area during an STW pause. However, the `gencon` global cycle, which runs when the *tenure* area is close to full, is split into three increments. The initial and final global cycle increments run during a relatively short STW pause. The intermediate global cycle increment, which consists of the majority of the GC cycle's work, runs its GC operations concurrently.
+
+Splitting the global cycle operations into these increments reduces pause times by running most of the GC operations concurrently with application threads. The `gencon` global cycle's concurrent increment is paused when a `gencon` partial GC cycle is triggered and resumes when the partial cycle(s) completes. In this way,a global cycle can progress while other types of cycle are run by pausing and resuming the concurrent work. In some policies, concurrent operations are split further into *multiple* concurrent increments for better control of progress of the concurrent operation.
+
+You can see this interleaving of the increments in the verbose GC log. The following table illustrates how the different increments of an example log interleave for the `gencon` policy (for clarity, not all GC activities are listed):
+
+<table style="width:100%">
+
+  <tr>
+    <th>Line Number</th>
+    <th colspan="2">Logging</th>
+  </tr>
+  <tr>
+    <th>-</th>
+    <th>`gencon` global GC cycle status</th>
+    <th>`gencon` partial GC cycle status</th>
+  </tr>
+  <tr>
+    <td>1-87</td>
+    <td colspan="2">initialization</th>
+  </tr>
+  <tr>
+    <td>87-51676</td>
+    <td> - </td>
+    <td> series of `gencon` partial cycles start and finish </td>
+  </tr>
+
+  <tr>
+    <td>51677</td>
+    <td> global cycle's initial increment marked</td>
+    <td>-</td>
+  </tr>
+
+  <tr>
+    <td>51680</td>
+    <td> STW pause starts</td>
+    <td>-</td>
+  </tr>
+
+  <tr>
+    <td>51683</td>
+    <td> global cycle starts</td>
+    <td>-</td>
+  </tr>
+
+    <tr>
+    <td>51684</td>
+    <td> STW pause ends</td>
+    <td>-</td>
+  </tr>
+
+  <tr>
+    <td>51709</td>
+    <td> concurrent increment runs (blank line)</td>
+    <td>-</td>
+  </tr>
+  <tr>
+    <td>51686</td>
+    <td> concurrent increment paused</td>
+    <td>STW pause starts</td>
+  </tr>
+    <tr>
+    <td>51690</td>
+    <td> concurrent increment paused</td>
+    <td>partial cycle starts</td>
+  </tr>
+
+  <tr>
+    <td>51708</td>
+    <td> concurrent increment paused</td>
+    <td>scavenge increment runs</td>
+  </tr>
+
+  <tr>
+    <td>51730</td>
+    <td> concurrent increment paused</td>
+    <td>partial cycle ends</td>
+  </tr>
+
+  <tr>
+    <td>51733</td>
+    <td> concurrent increment resumes</td>
+    <td>STW pause ends</td>
+  </tr>
+
```

Comment:
since you have '(blank line)' up above, it would be nice to mention it here too. don't know though if it would be good to associate with existing lines 'resume' or 'finishes', or to introduce new line 'runs'

Reply:
'blank line' removed from table to make this example relevant to all policies.  Resume and finish of concurrent increment is sometimes recorded in the logs.